Packs/illuminate/Integrations/Analyst1/Analyst1.yml

```diff
@@ -5,24 +5,30 @@ name: Analyst1
 display: Analyst1
 category: Data Enrichment & Threat Intelligence
 description: This integration utilizes Analyst1's system to enrich Demisto indicators with data provided by the Analyst1 REST API, such as actor and malware information, activity and reported dates, evidence and hit counts, and more.
+detaileddescription: |-
+  ### Partner Contributed Integration
+  #### Integration Author: Analyst1
+  Support and maintenance for this integration are provided by the author. Please use the following contact details:
+  - **Email**: [support@analystplatform.com](mailto:support@analystplatform.com)
+  ***
+  [View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/analyst1)
 configuration:
 - display: Analyst1 API Credentials (username/password)
   name: credentials
-  defaultvalue: ''
   type: 9
   required: true
-- display: Domain of Analyst1 server to use (format - server.thatyouuse.com - No http/https)
+- display: Domain of Analyst1 server to use (format - server.that_you_use.com - No http/https)
```

Comment:
```suggestion
- display: Domain of Analyst1 server to use
  additionalinfo: Format - server.that_you_use.com - No http/https
```
@a1-james What do you think about it?
This will solve the IN135 error in validate
![SCR-20230716-jxay](https://github.com/demisto/content/assets/90556466/dc763ebd-ba00-4b44-8ffb-eb1e7b71b854)